src/mrinufft/operators/interfaces/gpunufft.py

```diff
@@ -481,6 +500,23 @@ def uses_sense(self):
         """Return True if the Fourier Operator uses the SENSE method."""
         return self.raw_op.uses_sense
 
+    @FourierOperatorBase.samples.setter
+    def samples(self, samples):
+        """Set the samples for the Fourier Operator.
+
+        Parameters
+        ----------
+        samples: np.ndarray
+            The samples for the Fourier Operator.
+        """
+        if self.density_method is not None:
+            self.compute_density(self.density_method)
+        self.raw_op.set_pts(
+            samples,
+            density=self.density,
+        )
```

Comment:
At initialization this may triggers errors (raw_op might not be defined), and if it is defined, this does an extra computation.  
Maybe a `.update_samples` would be more convenient.